code/modules/surgery/tools.dm

```diff
@@ -239,3 +239,62 @@
 	else
 		to_chat(user, "[src] is empty.")
 	return
+
+/// SYNDICATE VARIANTS
+
+/obj/item/retractor/syndicate
+	name = "syndicate retractor"
+	desc = "Reinforced with carbon-plasma alloy for a stalwart grip."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "sretractor"
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3
+
+/obj/item/hemostat/syndicate
+	name = "syndicate hemostat"
+	desc = "Comes with carp-style teeth for complete bleeding control."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "shemostat"
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3
+
+/obj/item/cautery/syndicate
+	name = "syndicate cautery"
+	desc = "Uses state-of-the-art heating techniques to bring the pain!"
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "scautery"
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3
+
+/obj/item/surgicaldrill/syndicate
+	name = "syndicate surgical drill"
+	desc = "Outfitted with plasmic nanotubes in the bit and a suped-up motor."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "sdrill"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	force = 18
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3
+
+/obj/item/scalpel/syndicate
+	name = "syndicate scalpel"
+	desc = "It incises effortlessly with it's fixium blade."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "sscalpel"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	force = 15
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3
+
+/obj/item/circular_saw/syndicate
+	name = "syndicate circular saw"
+	desc = "It has a little sub-sentient AI on it to correct errors in the cutting process and guide the blade for extremely efficient cuts."

```

Comment:
This is getting a bit too extreme, too quickly.

Reply:
```desc = "Modified with a guidance computer for superhuman precision."```

Reply:
jjr is corect